tests/fixtures/populate_workspace_data.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+"""
+Upload fixtures into Terra data model
+"""
+import os
+import sys
+import json
+
+from firecloud import fiss
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import tests.config
+WORKSPACE_GOOGLE_PROJECT = os.environ['GOOGLE_PROJECT']
+WORKSPACE_NAME = os.environ['WORKSPACE_NAME']
+
+def list_entities():
+    resp = fiss.fapi.get_entities_with_type(WORKSPACE_GOOGLE_PROJECT, WORKSPACE_NAME)
+    resp.raise_for_status()
+    for ent in resp.json():
+        yield ent
+
+def delete_entities(entities):
+    resp = fiss.fapi.delete_entities(WORKSPACE_GOOGLE_PROJECT, WORKSPACE_NAME, entities)
+    resp.raise_for_status()
+
+def delete_all_entities():
+    entities_to_delete = [dict(entityType=e['entityType'], entityName=e['name'])
```

Comment:
Should this yield each element to a delete single item function instead of building the entire dictionary at once?

Reply:
This deletes the entire contents of the workspace, so I think it's OK for now.